### ISSUE TITLE
Adding a collapse section for all the contributors in the mentorship page

### ISSUE THREAD

**Issue by robin025** [chaoss/website#585]
hey, @klumb I think this small enhancement can be helpful for the website, Please have a look!
Right now in the [mentorship section](https://chaoss.community/chaoss-mentorship/) of the website has a table listing all the contributors that have contributed to this organization through many programs, but the links to the individual contributor section in the table section will only work in Github markdown, but not in the website
For Example : 
> |[Ria Gupta](#Ria-Gupta)|2020|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
above line of code will not interlink the [**#Ria-Gupta**](https://github.com/chaoss/website/blob/master/Community/alumni.md#ria-gupta) section on website, it will only add **#Ria-Gupta** to the suffix of the URL

**Comment by GeorgLink** [comment]
Thank you for spotting the malfunctioning links to anchors on the page.

I agree that this needs to be fixed.

An easy fix would be to update the links so that they work on the website only. The markdown file is not intended to be viewed on GitHub but only on the website, so breaking the GitHub UI experience is okay.

I also see the merit in making the sections collapsible. My concern with that is that the project titles are hidden behind a click. I like the table at the top because it shows the student's name, the project they worked on, when they worked on it, and who the mentors are -- three important pieces of information in my opinion. 

Right now, I would prefer to fix the links but I can be swayed.

**Comment by klumb** [comment]
There is some duplication in the way we are presenting the mentees and the projects. I like the drop down idea but the way it is working now it basically creates a list that looks just like the table.  

Can we merge these two areas together? The drop down would give us the ability to remove the table all together if presented well.

Or perhaps change the lower area to focus on the project rather than the mentee?

**Comment by klumb** [comment]
We can try to use the section tag but there is no guarantee that it will work. 

I don't believe that the GitHub Markdown element we use on WordPress allows full HTML5.

In the past when we have tried to use more advanced HTML5 methods and tags, they have been stripped out or converted to text.

I am comfortable experimenting with this but be aware that it may not work as a markdown file (which is our preferred form).

**Comment by robin025** [comment]
Hey @klumb  and @GeorgLink 
I was referring these two ([metrics-common.md](https://github.com/chaoss/website/blob/master/release/metrics-tables/metrics-common.md) and [metrics-table.md](https://github.com/chaoss/website/blob/master/release/metrics-tables/metrics-table.md)) files from metrics section and these two Markdown files are linking the sections correctly on website
One more thing, I have also gone through there Markdown code and found that each section is being assigned a link with prefix **#user-content**
![Screenshot (408)](https://user-images.githubusercontent.com/55009875/113326565-762bd300-9337-11eb-9930-716caec714e5.png)
So the question I have is how these two markdown files is being working properly, like is there any feature on WordPress where you have configured something where WordPress automatically adds **#user-content** to the ID attribute of anchor tag (in HTML) and then automatically fetches the following part to interlink the links
![Screenshot (409)](https://user-images.githubusercontent.com/55009875/113328486-f94e2880-9339-11eb-83fd-db34b6c95706.png)

Because I have also inspected the website and found that HREF attribute to that section is simple **#focus-area---what**, but the ID attribute to that section is this which you can find in the below image
![Screenshot (410)](https://user-images.githubusercontent.com/55009875/113329384-1fc09380-933b-11eb-95a4-e6506d67b306.png)

So at the last, my question is have you typed this ID attributes manually, or is it generated automatically with the help of any WordPress feature
Kindly review this and help me so that I can make a PR solving this issue asap!

**Comment by klumb** [comment]
I'm sorry, I don't understand the question.

Those links are working as intended. 

This is basic functionality of Markdown and HTML.

**Comment by robin025** [comment]
Usually the link in Markdown is written in this format where the title will be inserted in a square bracket **[]** and followed by the  link in parentheses **()**
For example  consider these 2 file ([metrics-common.md](https://github.com/chaoss/website/blob/master/release/metrics-tables/metrics-common.md) and [metrics-table.md](https://github.com/chaoss/website/blob/master/release/metrics-tables/metrics-table.md)) 
In metrics-table.md file link to the section **What - contributions are being made** is given by ``#user-content-focus-area---what`` as you can see in the following ![img](https://user-images.githubusercontent.com/55009875/113326565-762bd300-9337-11eb-9930-716caec714e5.png) and the data related to this link(#user-content-focus-area---what) is presented in metrics-common.md file with just a simple title **Focus Area - What** ![1](https://user-images.githubusercontent.com/55009875/113328486-f94e2880-9339-11eb-83fd-db34b6c95706.png)
So basically  when this markdown will be rendered on the website, after clicking to this link, it will scroll you down to that section
But after seeing the markdown of metrics-common.md file, there is no occurrence of **#user-content** keyword, so how these two files are working properly

**Comment by klumb** [comment]
This is how it works 

Example:
to link to ###My Heading

The syntax could be
[My Heading](#my-heading)
or
[My Heading](#user-content-my-heading)

Anchor IDs are assigned to all headings when rendered as HTML. The ID is the heading text, with the spaces replaced by dashes (-) and all lower case.

**Comment by robin025** [comment]
> This is how it works
> 
> Example:
> to link to ###My Heading
> 
> The syntax could be
> [My Heading](#my-heading)
> or
> [My Heading](#user-content-my-heading)
> 
> Anchor IDs are assigned to all headings when rendered as HTML. The ID is the heading text, with the spaces replaced by dashes (-) and all lower case.

Okay @klumb got it, actually I was not aware of this **[My Heading] (#user-content-my-heading)**
but if this is the case then why links of contributors in [mentorship page](https://chaoss.community/chaoss-mentorship/) is not working, as Markdown in [alumini.md](https://raw.githubusercontent.com/chaoss/website/master/Community/alumni.md) is also correct or should we try to use this  **[My Heading] (#user-content-my-heading)**
